src/main/java/ch/njol/skript/expressions/ExprWithFireResistance.java

```diff
@@ -0,0 +1,88 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("With Fire Resistance")
+@Description({
+	"Creates a copy of an item with (or without) fire resistance.",
+	"Note: items that already have fire resistant (e.g. netherite armour, tools, etc) currently can't have their fire resistance removed.",
+	"This was missed in 1.20.5 and .6, but should be fixed in 1.21."
```

Comment:
This may be my opinion, and probably up for debate, but I feel this line shouldn't be here.
While we do have a snapshot for 1.21, things may change, and we can't guarantee the outcome.
This will also look confusing down the road (lets pretend we're on 1.25)

Reply:
If you're speaking in reference of the fact netherite items do not work I personally say it's best for it to exist now than not. I'd much rather have the ability to point towards docs than see an issue opened by someone say "I can't make my prot 1001 netherite helmet be burned by lava"

```suggestion
	"Currently in 1.20.6 netherite items can't be manipulated, however this will be changed in 1.21"
```
How does this sound instead? As they do intend to make components removable and from my testing it does work on netherrite items